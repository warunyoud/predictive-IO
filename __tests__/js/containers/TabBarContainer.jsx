import React from 'react';
import TabBarContainer from '../../../src/js/containers/TabBarContainer';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer
    .create(<TabBarContainer
      tabs={[{ index: 0, subIndex: -1, text: 'hello' }]}
      setCurrentTabIndex={ jest.fn()}
      currentTabIndex={0}
      />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
