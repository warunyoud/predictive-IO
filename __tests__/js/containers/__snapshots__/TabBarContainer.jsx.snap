// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="tab-bar"
>
  <div
    className="tab-item tab_italic"
    onClick={[Function]}
  >
    hello
  </div>
</div>
`;
